pyrasterframes/src/main/python/docs/raster-read.pymd

```diff
@@ -180,6 +184,29 @@ mb2 = spark.read.raster(catalog=spark.createDataFrame(mb_cat),
 mb2.printSchema()
 ```
 
+## GeoTrellis
+
+### GeoTrellis Catalogs
+
+[GeoTrellis][GeoTrellis] is one of the key libraries that RasterFrames builds upon. It provides a Scala language API to working with large raster data with Apache Spark. RasterFrames provides a DataSource that supports both reading and @ref:[writing](raster-write.md#geotrellis-layers) with GeoTrellis.
+
+A GeoTrellis catalog is a set of GeoTrellis layers. We can read a dataframe giving details of the content of a catalog using the following. The scheme is typically `hdfs` or a cloud storage provider like `s3` or `wasb`.
+
+```python, evaluate=False
+gt_cat = spark.read.format('geotrellis-catalog').load('scheme://path-to-gt-catalog')
```

Comment:
We should create an extension method for this to be consistent with other usesages.